### PR TITLE
[EXPO] Fix `expo web` aliase to `expo start:web`

### LINKED ISSUE #159: `expo web` command not recognized
`expo web` is an unrecognized alias for `expo start:web`

Matt tells me it should be a simple fix: just finding the `start:web` subcommand and then convert the `name` property from a string to an array ("start:web"  →  ["start:web", "web"])!

### PR DESCRIPTION
**Pull request by QuiiBz**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Fixes #159 

**What is the current behavior? (You can also link to an open issue here)**
`expo web` is not recognized

**What is the new behavior (if this is a feature change)?**
`expo web` is recognized

**Additional info:**
Screenshot 2021-05-02 at 08 37 44
I also removed all empty arrays of `subcommands` since that wasn't needed.

### CODE REVIEW

File: dev/expo.ts [hunk 1]
```diff
@@ -1,3 +1,89 @@
...
+    args: {},
+  },
+  {
+    name: ["--config"],
+    description: "Specify a path to app.json or app.config.js\n",
+    args: {},
+  },
+  {
+    name: ["-h", "--help"],
+    description: "output usage information\n  \n  \n",
```
Comment (cstrnt) [inline dev/expo.ts:82]: I think we don't need the `\n`s here

File: dev/expo.ts [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+  {
+    name: ["-w", "--web"],
+    description: "Opens your app in a web browser\n",
+    args: {},
+  },
+  {
+    name: ["-m", "--host"],
+    description:
+      'lan (default), tunnel, localhost. Type of host to use. "tunnel" allows you to view your link on other networks\n',
+    args: {},
```
Comment (cstrnt) [inline dev/expo.ts:53]: Wouldn't it make sense to add `tunnel` `lan` and `localhost` as suggestions here?

**Review by cstrnt — CHANGES_REQUESTED** [review]

**Comment by QuiiBz** [comment]
I copy/pasted the existing spec, didn’t noticed that there were some issues in it. Let me fix them

**Comment by cstrnt** [comment]
Thank you very much, Tom 😊

File: dev/expo.ts (hunk 1 above)
Comment (QuiiBz) [reply dev/expo.ts]: Fixed thanks to the power of regex  😄

File: dev/expo.ts (hunk 2 above)
Comment (QuiiBz) [reply dev/expo.ts:52]: Done, looks cool!
Screenshot 2021-05-02 at 12 51 52

**Review by mattschrage — DISMISSED** [review]
Love the addition of the `host` suggestions and thanks for stripping the newline literals!

You can add an alias to a subcommand just by include an array of names in the `name` prop.

So `name: "web"` can become `name: ["start:web", "web"]` 

This just saves you from needing to extract all of the shared options and is a little cleaner! We might need to document this a bit better 😅

**Comment by QuiiBz** [comment]
> Love the addition of the `host` suggestions and thanks for stripping the newline literals!
> 
> You can add an alias to a subcommand just by include an array of names in the `name` prop.
> 
> So `name: "web"` can become `name: ["start:web", "web"]`
> 
> This just saves you from needing to extract all of the shared options and is a little cleaner! We might need to document this a bit better 😅

I see! But currently, the `Subcommand` interface only accepts a string. We will need to update this first in order to change this spec:
Screenshot 2021-05-04 at 09 33 58
Based on. what you said, the type of `name` should be changed to: `SingleOrArray<string>`

**Comment by cstrnt** [comment]
@mattschrage this is new, right? Because I remember bumping into that problem and it didn't work back then.

@QuiiBz I think so!

**Comment by mattschrage** [comment]
> @mattschrage this is new, right? Because I remember bumping into that problem and it didn't work back then.

I just tested it and you're totally right - having an array for the `name` field in a Subcommand isn't currently supported. cc @falky97

**Comment by falky97** [comment]
^^ Yes this is fixed as matt outlined above

**Comment by QuiiBz** [comment]
Sweet! Just updated, ready to merge in.

**Review by cstrnt — APPROVED** [review]

**Comment by cstrnt** [comment]
Well done, Tom

**Pull request merged**